.github/workflows/app-react.yml

```diff
@@ -0,0 +1,30 @@
+name: APP (React)
```

Comment:
Should this file be renamed to better reflect the tech stack used in the project?

Reply:
yes